code/modules/lighting/lighting_object.dm

```diff
@@ -139,7 +133,11 @@ GLOBAL_LIST_EMPTY(all_lighting_objects) // Global list of lighting objects.
 /atom/movable/lighting_object/blob_act()
 	return
 
+// Nope nope nope!
+/atom/movable/lighting_object/onShuttleMove(turf/T1, rotation)
```

Comment:
I'm guessing this just changes it to return FALSE instead of null but I'm not sure